packages/drivers/debugger/tsconfig.json

```diff
@@ -7,9 +7,18 @@
     "compilerOptions": {
         "rootDir": "./src",
         "outDir": "./dist",
-        "lib": [ "ES2017", "ES2018.Promise", "ES2018.AsyncIterable", "DOM", "DOM.Iterable" ]
+        "lib": [
+            "ES2017",
+            "ES2018.Promise",
+            "ES2018.AsyncIterable",
+            "DOM",
+            "DOM.Iterable"
+        ],
+        "types": [
```

Comment:
bohemia consumes this package so we shouldn't add node types here (to make it harder to accidentally introduce them in the code)

Reply:
Reverted.